src/main.js

```diff
@@ -190,6 +193,21 @@ function configureCommandlineSwitchesSync(cliArgs) {
 				}
 			}
 
+			// Locale
+			else if (argvKey === 'locale') {
+				if (product.quality === 'code-insiders') {
```

Comment:
```suggestion
				if (product.quality === 'insider') {
```

Refs https://github.com/microsoft/vscode-distro/blob/164595434a0f944feb5cfa6202186b68dade393c/quality/insider/product.json#L43

Reply:
Alternatively `product.quality !== 'stable'` to support our exploration builds too.

Reply:
@bpasero can we instead have `product.quality === 'insider' || product.quality === 'exploration'` since we also don't want to enable this change in OSS till we update to Electron v21